client/profile-wizard/steps/store-details.js

```diff
@@ -163,76 +172,117 @@ class StoreDetails extends Component {
 
 	render() {
 		const { showUsageModal } = this.state;
+		const { skipProfiler } = this.props;
 
 		return (
 			<Fragment>
 				<H className="woocommerce-profile-wizard__header-title">
-					{ __( 'Where is your store based?', 'woocommerce-admin' ) }
+					{ __( 'Welcome to WooCommerce', 'woocommerce-admin' ) }
 				</H>
 				<H className="woocommerce-profile-wizard__header-subtitle">
 					{ __(
-						'This will help us configure your store and get you started quickly',
+						"Tell us about your store and we'll get you set up in no time",
 						'woocommerce-admin'
 					) }
-				</H>
 
-				<Card>
-					<Form
-						initialValues={ this.initialValues }
-						onSubmitCallback={ this.onSubmit }
-						validate={ validateStoreAddress }
+					<Tooltip
+						text={ __(
+							'Your store address will help us configure currency\n options and shipping rules automatically.\n This information will not be publicly visible and can\n easily be changed later.',
+							'woocommerce-admin'
+						) }
```

Comment:
We need to supply `aria-label` as well since this component is invisible to screen readers. See the "accessibility note" here: https://github.com/WordPress/gutenberg/tree/master/packages/components/src/tooltip#tooltip

Reply:
Good point, thanks. Will fix.